### PR TITLE
adding CriteriaThumbnail and Thumbnail class

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**
OK Team, please check my code

### CODE REVIEW

**Comment by alszla** [comment]
OK

File: src/main/java/com/fundation/webservice/model/ThumbnailVideo.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+/**
+ * Implements Thumbnail class an convert Method
+ *
+ * @author Josue Rodriguez
+ * @version 1.0
+ */
+public class ThumbnailVideo {
+    CriteriaThumbnailVideo criteria;
+    public ThumbnailVideo(CriteriaThumbnailVideo criteria) {
+        this.criteria =  criteria;
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ThumbnailVideo.java]: please remove the additional space after "= criteria"

File: src/main/java/com/fundation/webservice/model/ThumbnailVideo.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        try{
+            String cmd = "ffmpeg -i "
+                    + criteria.getSrcPath()
+                    + " -ss "
+                    + criteria.getTime()
+                    + " -vframes 1 -s 128x128 "
+                    + criteria.getDestPath()
+                    + criteria.getName()+"."
+                    + criteria.getExt();
+            Process p = Runtime.getRuntime().exec(cmd);
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ThumbnailVideo.java]: improve "p" variable, it is don't understand what is their function.

File: src/main/java/com/fundation/webservice/model/ThumbnailVideo.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            BufferedReader in = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+            String line ;
+            while((line = in.readLine()) != null){
+                System.out.println(line);
+            }
+            p.waitFor();
+            in.close();
+        }catch(Exception e){
+
```
Comment (LimbertVargas) [inline src/main/java/com/fundation/webservice/model/ThumbnailVideo.java]: It is necessary this space?

**Comment by LimbertVargas** [comment]
It's all ok Josue, good job.

File: src/main/java/com/fundation/webservice/model/ThumbnailVideo.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+                    + criteria.getSrcPath()
+                    + " -ss "
+                    + criteria.getTime()
+                    + " -vframes 1 -s 128x128 "
+                    + criteria.getDestPath()
+                    + criteria.getName()+"."
+                    + criteria.getExt();
+            Process p = Runtime.getRuntime().exec(cmd);
+            BufferedReader in = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+            String line ;
```
Comment (alszla) [inline src/main/java/com/fundation/webservice/model/ThumbnailVideo.java]: Line 37 and the "while" block are not actually used.

**Pull request merged**